src/pages/CardAlias/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from 'react-router-dom';
+
+import Card from '../../components/Card';
+import useInput from '../../hooks/useInput';
+import { isValidCardAlias } from '../AddCard/domain/validation';
+import { registerCardAlias } from '../../utils/card';
+import './index.css';
+import useRedirection from '../../hooks/useRedirection';
+import useLocalStorage from '../../hooks/useLocalStorage';
+
+const CardAliasPage = () => {
+  const navigate = useNavigate();
+  const { value, onChange } = useInput(isValidCardAlias);
+  const { value: cardList, postLocalStorage } = useLocalStorage('cardList', '[]');
+  const currentCard = cardList[cardList.length - 1];
+
+  useRedirection(cardList.length === 0 || currentCard.alias);
+
+  const onConfirmButtonClick = () => {
+    postLocalStorage(registerCardAlias(cardList, value, currentCard.cardNumber));
+    navigate('/');
+  };
+
+  return (
+    <div className="card-alias-page">
+      <h2>카드 등록이 완료되었습니다.</h2>
+      <Card
+        cardCompany={currentCard.cardCompany}
+        cardFirstNumber={currentCard.cardNumber.first}
+        cardSecondNumber={currentCard.cardNumber.second}
+        cardThirdNumber={currentCard.cardNumber.third}
+        cardFourthNumber={currentCard.cardNumber.fourth}
+        cardOwner={currentCard.cardOwner}
+        expireMonth={currentCard.expireMonth}
+        expireYear={currentCard.expireYear}
+      />
```

Comment:
```suggestion
      <Card
        {...currentCard} 
        cardFirstNumber={currentCard.cardNumber.first}
        cardSecondNumber={currentCard.cardNumber.second}
        cardThirdNumber={currentCard.cardNumber.third}
        cardFourthNumber={currentCard.cardNumber.fourth}
      />
```

객체는 2뎁스 이하까지 내려가면 직관성이 많이 떨어지고 핸들링이 복잡해진답니다🥲

Reply:
확실히 depth가 깊어지니 props를 넘겨주거나 핸들링하는 과정에서 어려워지더라구요..! 시간이 남으면 개선해보겠습니다!